apps/src/music/player/MusicPlayer.ts

```diff
@@ -539,7 +539,7 @@ export default class MusicPlayer {
       return;
     }
 
-    for (const instrument of library.instruments.filter(
+    for (const instrument of [...library.instruments, ...library.kits].filter(
       folder => folder.type === 'instrument' || folder.type === 'kit'
     )) {
```

Comment:
Keeping in mind I know nothing about the data, but why is this filter necessary?

Just looking at it, it looks like it's iterating over a list of instruments and a list of kits, but then it's filtering out any folder type that isn't an instrument or a kit.

Which implies that we could have something that's not an instrument or a kit in the list of instruments and kits.

How could that happen? Is it something that should/could be prevented elsewhere?

Reply:
Good catch! The `filter` is old code from when we didn't separate `instrument`s and `kit`s in the library. The filter can be removed now.

Reply:
Thanks for the catch.  It looks like this code was removed entirely in https://github.com/code-dot-org/code-dot-org/pull/57849?